src/scmdata/dataframe.py

```diff
@@ -1233,49 +1233,39 @@ def convert_unit(
             return ret
         return None
 
-    def relative_to_ref_period_mean(
-        self, append_str: Optional[str] = None, **kwargs: Any
-    ) -> pd.DataFrame:
+    def relative_to_ref_period_mean(self, append_str=None, **kwargs):
         """
         Return the timeseries relative to a given reference period mean.
 
         The reference period mean is subtracted from all values in the input timeseries.
 
         Parameters
         ----------
-        append_str
```

Comment:
Keep the docs, but mark as not used. Perhaps a warning if it is not None, to help find usages

Reply:
What does mark as not used mean? Is that a code thing or a docs thing?

Reply:
Perhaps change to 
```
Deprecated. A :exc:`NotImplementedError` is raised if :obj:`append_str` is not `None`.
```

Reply:
beauty